### PR TITLE
tempest suite to run on all SKUs

### PR DESCRIPTION
**Pull request by vganapath**

### CODE REVIEW

File: contrail/functions [hunk 1]
```diff
@@ -51,9 +51,9 @@ function get_image_id() {
     else
         image_filename=$(basename $image_path)
         if [ ! -f $image_filename ]; then
-            `wget $image_path --no-check-certificate`
+            `wget $image_path`
```
Comment (vedujoshi) [inline contrail/functions]: It would be good to have no-check-certificate in case there is a https url ?

File: contrail/functions [hunk 2]
```diff
@@ -51,9 +51,9 @@ function get_image_id() {
...
         fi
-        export OS_IMAGE_URL=http://${keystone_host}:9292/ && ${with_venv} glance image-create --name $image_name --disk-format qcow2 --container-format bare  --file $image_filename --visibility public > /dev/null
+        export OS_IMAGE_URL=http://${keystone_host}:9292/ && ${with_venv} glance image-create --name $image_name --disk-format qcow2 --container-format bare --is-public True --file $image_filename > /dev/null
```
Comment (vedujoshi) [inline contrail/functions]: is-public is not a argument in mitaka...

File: requirements.txt [hunk 3]
```diff
@@ -1,28 +1,33 @@
-# The order of packages is significant, because pip processes them in the order
```
Comment (vedujoshi) [inline requirements.txt]: We should get rid of requirements.txt like in upstream tempest 
It always introduced maintenance problems

File: tempest/services/identity/xml/identity_client.py [hunk 4]
```diff
@@ -0,0 +1,161 @@
+# Copyright 2012 IBM Corp.
```
Comment (vedujoshi) [inline tempest/services/identity/xml/identity_client.py]: I think we should not add any of these files on our own. Let tempest take care of it?

File: contrail/functions (hunk 2 above)
Comment (klash) [inline contrail/functions]: Try `--visibility public`, I know that works for liberty

File: contrail/functions [hunk 5]
```diff
@@ -1,6 +1,19 @@
...
 
+declare -A sku_dict
+sku_dict['2014.1']='icehouse'
+sku_dict['2014.2']='juno'
+sku_dict['2015.1.2']='kilo'
+sku_dict['12.0']='liberty'
+sku_dict['13.0']='mitaka'
+
+function get_sku() {
+    ver=`${with_venv} nova-manage version`
```
Comment (smurugap) [inline contrail/functions]: This adds a dependency that the tempest script has to be run from the openstack node.
Do we have a better option? Shall we use the client version, moreover its the cli option?

Also the mapping isnt right. The version output should be trimmed to single dotted version.
root@b3s10:~# ver=`nova-manage version`
root@b3s10:~# echo $ver
2014.2.3
root@b3s10:~# echo ${sku_dict[${ver}]}

root@b3s10:~#

Comment (smurugap) [inline contrail/functions]: BTW, i dont see the upstream changesets. I think this pull request is just the merge conflict resolution. Also the merge conflict resolution for test-requirements.txt is missing and run-contrail-tempest.sh is not complete.

**Pull request closed without merging**